views/js/controller/creator/views/property.js

```diff
@@ -129,17 +129,33 @@ function($, ui, DataBinder, templates){
         * @private
         */
         function propValidation() {
-            var $togglers;
+            const $togglers = $('#authoringBack, #saver, #previewer');
+
             $view.on('validated.group', function(e, isValid){
+                let classErrorAll = $container.find('span.validate-error');
+                let classErrorVisibleFields = $('.section-props:visible').find('span.validate-error');
+
                 if(e.namespace === 'group'){
-                    $togglers = $('#test-creator .property-toggler, #saver');
-                    if(isValid === true){
+                    let testSectionId = '#' + $('.tlb-button-on').parents('.section').attr('id');
+                    let $deleteButtonInCurrentSection = $(testSectionId).find('[data-testid="remove-section"]');
+
+                    if(isValid === true && classErrorVisibleFields.length === 0 ){
+                        $(testSectionId).removeClass('section-error');
                         $togglers.removeClass('disabled');
```

Comment:
can you move `$togglers.removeClass('disabled');` and `$togglers.addClass('disabled');` to the next `if (classErrorAll.length > 0 ){ ... else`
to have more clear logic